### ISSUE TITLE
Add option to disable instant alt-text generation on file upload

### ISSUE THREAD

**Issue by Idleworks** [marketing-factory/ai-filemetadata#39]
On some websites, editors may want to initiate the generation of alt texts themselves, e.g. to have the power to decide which images should be sent to the external service and which should not. 
Therefore there should be an option to disable automatic generation after file upload.
If an editor wants to use the service, they can click on the button next to the alt text field.
